src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_Hatch.java

```diff
@@ -79,6 +79,7 @@ public final void updateTexture(byte texturePage, byte machineBlock){
     @Override
     public final void onValueUpdate(byte aValue) {
         mMachineBlock=actualTexture=(byte)(aValue & 0x7F);
+        mTexturePage=0;
```

Comment:
in case someone uses onValueUpdate revert the page to 0
in the updated code -> the next line should change the page to x